README.md

```diff
@@ -2,58 +2,66 @@
+[Docker](https://docs.docker.com/get-docker/) is required to initialize, develop and build your package.
 
-## Create a custom package repository
+## Create a custom package template repository
 
-To create a component package, create a new repository in GitHub using this repository as a template.
+To create a package template, create a new repository in GitHub using this repository as a template.
 
 ![Use this template menu](docs/creation-1.png) ![Create new repository menu](docs/creation-2.png)
 
 Alternatively, you can also clone this repository locally and create a new repository from it.
 
 ```bash
-git clone git@github.com:aica-technology/component-template.git my_component_package
+git clone git@github.com:aica-technology/package-template.git my_package
 ```
 
-## Initialize the template component package
+## Running the package template wizard
 
-This template repository uses the placeholder package name `template_component_package` and includes four different
-component types: C++ component, C++ lifecycle component, Python component, and Python lifecycle component.
+When you first clone the template repository, no source will be populated. First, start the initialization wizard by
+executing `./initialize_templates.sh`. An interactive console UI will guide you through the necessary steps to populate
+the directory. During this process, you may opt in/out to specific templates or types of components to generate.
 
-The first step is to rename the package to a meaningful custom name and remove any unnecessary component types. You can
-accomplish this by using the included [`initialize_package`](./initialize_package.sh) script. In a terminal, run
+In case you already ran the wizard, but are not happy with your selection, you may re-run it as if it was the first run.
+**This will remove any files previously created through the wizard!**
 
-```bash
-./initialize_package.sh [--dry-run] [--help]
-```
+If you select more than one packages during the wizard run, the wizard will generate a _collection_ of packages. You
+will be asked for a collection name and the changes will be reflected in your `aica-package.toml`.
 
-from the top level directory of the repository. The script prompts you with the name for the package and the component
-types that should be included.
+If you want to include even more ROS packages within your collection, create a new package folder under `source` and add
+it to your `aica-package.toml` file under `[build.packages.name_of_new_package]` accordingly.
 
-If you want to include a second ROS package within the AICA package, create a new package folder in `source` and add it
-to your `aica-package.toml` file under `[build.packages.name_of_new_package]` accordingly.
+For the purposes of this README, we refer to both collection and (mono-)package outputs simply as "package". The only
+differences between them are the adjustments specified in `aica-package.toml`; otherwise, no additional steps are
+required.
```

Comment:
@eeberhard added this just make sure it's mentioned explicitly at least once. I otherwise changed phrasing a bit to mention everything as `package`. I think the `collection` was already used where specifically needed